### PR TITLE
Fix 'uploads/tmp/' dir creation, refs #11461

### PR DESCRIPTION
**Pull request by jraddaoui**
Make sure 'uploads/' is created first

### CODE REVIEW

**Comment by sbreker** [comment]
Looks good.  👍

File: lib/Qubit.class.php [hunk 1]
```diff
@@ -351,4 +346,24 @@ public static function moveUploadFile($file)
...
+   */
+  public static function createUploadDirsIfNeeded()
+  {
+    $uploadsPath = sfConfig::get('sf_upload_dir');
+    $directories = array(
+      $uploadsPath,
+      $uploadsPath . DIRECTORY_SEPARATOR . 'tmp'
+    );
+
+    foreach ($directories as $dirPath)
```
Comment (MikeFE) [inline lib/Qubit.class.php:361]: Could use mkdir(..., ..., true) instead of a loop here to create both folders I think.

Comment (jraddaoui) [reply lib/Qubit.class.php:361]: Yes, but someone was complaining about some issues with the permissions on Windows, so I decided to leave it like that, at least we're not calling chmod(), heheh

Comment (sbreker) [reply lib/Qubit.class.php:361]: With this implementation if the need arose, we could add additional folders to the array.  🤔

**Pull request merged**